bin/mdsctl

```diff
@@ -65,8 +82,12 @@ declare -A configs=(
   ["setfiles"]="${MDS_SET_FILES:-}"
   ["setfiles-mds"]="${MDS_SET_FILES_MDS:-}"
   ["namespace"]="${MDS_NAMESPACE:-default}"
-  ["namespace-mds"]="${MDS_NAMESPACE_MDS}"
-  ["dryrun"]="${MDS_DRY_RUN:-}")
+  ["namespace-metrics-adapter"]="${MDS_NAMESPACE_METRICS_ADAPTER:-util}"
+  ["namespace-dashboard"]="${MDS_NAMESPACE_LOGGING:-dashboard}"
+  ["namespace-logging"]="${MDS_NAMESPACE_LOGGING:-util}"
+  ["namespace-mds"]="${MDS_NAMESPACE_MDS:-mds}"
+  ["dryrun"]="${MDS_DRY_RUN:-}"
+  ["quiet"]="${MDS_QUIET:-}")
```

Comment:
@drtyh2o an existing-and-non-empty env var MDS_QUIET will effectively inline '--quiet' for every invocation, eg:

`export MDS_QUIET=true`